### PR TITLE
create sealed secret script for IBM Cloud API key

### PR DESCRIPTION
**Pull request by jeffdmgit**
This also includes the SealedSecret yaml for Allan's IBM Cloud API Key.

### CODE REVIEW

File: operators/api-key-sealedsecret/api-key-sealedsecret.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```
Comment (Mattar555) [inline operators/api-key-sealedsecret/api-key-sealedsecret.sh]: Looks good, but this is the wrong location. It should be in the services/instances (this is not an operator but an instance of the sealed secret operator which was already created)

Don't forget to make an app in otp-gitops too!

Comment (jeffdmgit) [reply operators/api-key-sealedsecret/api-key-sealedsecret.sh]: Moved: 
```
 renamed:    operators/api-key-sealedsecret/api-key-sealedsecret.sh -> instances/api-key-sealedsecret/api-key-sealedsecret.sh
 renamed:    operators/api-key-sealedsecret/api-key-secret.yaml -> instances/api-key-sealedsecret/api-key-secret.yaml
```

Comment (jeffdmgit) [reply operators/api-key-sealedsecret/api-key-sealedsecret.sh]: Added the app in otp-gitops: https://github.com/oto-cp4ba-gitops/otp-gitops/pull/14

not sure if that's correct, however.

Comment (Mattar555) [reply operators/api-key-sealedsecret/api-key-sealedsecret.sh]: Correct.

I added the Kustomize entry too. Thanks and well done

**Pull request closed without merging**